src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1809,11 +1809,32 @@
       <Kind Name="SemicolonToken"/>
     </Field>
   </Node>
+
+  <Node Name="VariableDeconstructionDeclaratorSyntax" Base="CSharpSyntaxNode">
+    <Kind Name="VariableDeconstructionDeclarator"/>
+    <Field Name="OpenParenToken" Type="SyntaxToken">
+      <Kind Name="OpenParenToken"/>
+    </Field>
+    <Field Name="Variables" Type="SeparatedSyntaxList&lt;VariableDeclarationSyntax&gt;"/>
+    <Field Name="CloseParenToken" Type="SyntaxToken">
+      <Kind Name="CloseParenToken"/>
+    </Field>
+    <Field Name="EqualsToken" Type="SyntaxToken" Optional="true"> <!-- PROTOTYPE(tuples) should be Opt -->
+      <Kind Name="EqualsToken"/>
+    </Field>
+    <Field Name="Value" Type="ExpressionSyntax" Optional="true"/> <!-- PROTOTYPE(tuples) should be Opt -->
```

Comment:
The comment here is to add "Opt" to the field names (which I delayed because it caused some surprising source generator issue).
If I understood your comment, you're saying "Opt" suffix is not required?